Sockets/Sockets.Implementation.NET/TcpSocketClient.cs

```diff
@@ -55,7 +87,14 @@ internal TcpSocketClient(TcpClient backingClient)
         /// </summary>
         public Stream ReadStream
         {
-            get { return _backingTcpClient.GetStream(); }
+            get
+            {
+                if (_secureStream != null)
```

Comment:
Yes, again, this IF was made on a last clean up and I forgot to set to true. Can you change it or should a I do it and push again? (git noob)